notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>After your model has finished training, you can review <strong>it's evaluation scores</strong>. </p><p> </p><p><em>The code below appears to be listing model evaluations, not models, yet this text block talks about listing models. Which is correct? Also I suggest not explaining an option not shown in the code below.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>